### PR TITLE
Conference capstone

### PR DESCRIPTION
**Pull request by akos29**
### Hello Microverse

In this pull request I included the following:

- Create a mobile part of the home and about webpages 
- Add the desktop version and the interaction part of the webpages 
- Add JavaScript for the menu and generate HTML on the fly.
- Update the styling

### CODE REVIEW

**Review by Bondok6 — COMMENTED** [review]
## Hi @akos29 🙋‍♂️, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [x] You have submitted a project, that doesn't have Linters, kindly add `.github/workflows` folders and `linters.yml` file to setup Linters correctly
> For this project, you should use [html-css-js](https://github.com/microverseinc/linters-config/tree/master/html-css-js) Linters.
1. Linter should be set up correctly for this repository.
1. Linter result should be green, this means that the last commit have a green check mark (✅)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @akos29 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request :heavy_multiplication_x: 
Invalid

You have submitted a project, that: 
- [x] Is not documented in a professional way. `Kindly make sure you give appropriate credit to the author in your README as said in the project requirements`
- [x] `Kindly make sure you have deployed your application on GitHub pages with the correct branch(conference-capstone)`
- [x] `Kindly make sure you include the link to your loom video presenting your project in your README or here in the PR summary`

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: assets/js/main.js [hunk 1]
```diff
@@ -1,8 +1,85 @@
...
+toggler.addEventListener('click', toggle);
+
+document.querySelectorAll('.nav-link').forEach((item) => {
+  item.addEventListener('click', toggle);
+});
+
+const speakersList = [
+  {
+    name: 'James M. Bing',
+    pp: './assets/images/speaker 2.jpg',
```
Comment (franclobo) [inline assets/js/main.js]: - [x] Please 🙏🏽 correct the path of the images to be able to see the images in the deployment 👇🏽 

image

File: assets/css/about.css [hunk 2]
```diff
@@ -3,26 +3,40 @@
...
+  padding: 7.75rem 2rem 5rem;
   background-color: var(--bs-gray);
   align-items: center;
   justify-content: center;
+  gap: 2rem;
+  background-image: url('../images/header.png');
+
+  /* background-repeat: no-repeat; */
+  background-size: cover;
 }
```
Comment (franclobo) [inline assets/css/about.css]: - [x] There are differences between the original design and yours in the size of this section, kindly correct the size to be similar to the original design 👇🏽 

| Original design | Your design |
|---|---|
| image | image |

File: about.html [hunk 3]
```diff
@@ -2,406 +2,109 @@
...
-				<p>2022 African Development Forum</p>
-			</div>
-		</footer>
-
-		<script
-			src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js"
-			integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3"
-			crossorigin="anonymous"
-		></script>
+			</footer>
```
Comment (franclobo) [inline about.html]: - [x] There are small differences in the footer of the mobile version, please 🙏🏽 correct this section on the home page and the about page to be similar to the original version.

image

File: about.html [hunk 4]
```diff
@@ -2,406 +2,109 @@
...
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g></svg
-						></span>
-						<h3>Exhibition</h3>
+				<span class="site-map">Home <span class="gt">&gt;</span> About</span>
+				<section class="show-case">
+					<h1 class="">African Development Forum Global Summit 2022</h1>
+					<div class="detail">
```
Comment (franclobo) [inline about.html:52]: - [x] Please correct the size of this section on the about page to be similar to the original version 👇🏽 

| Original version | Your design |
|---|---|
| image | image |

File: about.html [hunk 5]
```diff
@@ -2,406 +2,109 @@
...
 								<a class="nav-link active" aria-current="page" href="about.html"
 									>About</a
 								>
 								<a class="nav-link" href="#">Program</a>
 								<a class="nav-link" href="#">Join</a>
 								<a class="nav-link" href="#">Sponsor</a>
-								<a class="nav-link" href="#">News</a>
-								<a class="nav-link disabled">ADF Campaign</a>
+
+								<a class="nav-link btn">ADF Campaign</a>
```
Comment (franclobo) [inline about.html:44]: - [x] There are two navbars in the original design, please 🙏🏽 add the black navbar.

image

File: index.html [hunk 6]
```diff
@@ -2,45 +2,39 @@
...
+								>
+								<a class="nav-link" aria-current="page" href="about.html"
 									>About</a
 								>
 								<a class="nav-link" href="#">Program</a>
 								<a class="nav-link" href="#">Join</a>
 								<a class="nav-link" href="#">Sponsor</a>
-								<a class="nav-link" href="#">News</a>
+
 								<a class="nav-link btn">ADF Campaign</a>
```
Comment (franclobo) [inline index.html:44]: - [x] There are two navbars in the original design, please 🙏🏽 add the black navbar 👇🏽 

image

File: README.md [hunk 7]
```diff
@@ -1,18 +1,25 @@
...
+## Designed by 
+
+-  Cidny Shin is the one the credit goes to for creating such an elegant and simple design. You can refer the mockup [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
 ## Live Demo (if available)
 
 [Live Demo Link](https://akos29.github.io/conference/)
 
+## Loom Video
+
+- Please refer to my brief [demonstration](https://www.loom.com/share/fe99979688214cc8b3bd83a11b8214f9) about the website. Thank you in advance for watching.
```
Comment (franclobo) [inline README.md:22]: - [ ] Kindly highlight some interesting piece of code or something you built that you are very proud of.

File: about.html [hunk 8]
```diff
@@ -2,406 +2,109 @@
...
 	<body>
 		<div class="wraper">
 			<header>
-				<nav class="navbar navbar-collapse bg-light">
-					<div class="container-lg">
-						<a class="navbar-brand" href="index.html"
+				<nav class="menu-container">
+					<button class="menu-toggler" type="button">
+							<span class="menu-toggler-icon"><i class="fa fa-bars"></i></span>
+						</button>
```
Comment (franclobo) [inline about.html:25]: - [x] I couldn't see the hamburger button in the mobile version, please 🙏🏽 fix it.

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @akos29, 

# Required changes ♻️ 

Good job so far!
You are doing it well 👍🏽 there are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [x] I couldn't find where you defined the colors like `--bs-primary` or `--bs-secondary`, please 🙏🏽  use the colors of the original design in your CSS file.

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @akos29 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**